modules/redpanda/redpanda_test.go

```diff
@@ -282,6 +282,50 @@ func TestRedpandaWithTLS(t *testing.T) {
 	require.Error(t, results.FirstErr(), kerr.UnknownTopicOrPartition)
 }
 
+func TestRedpandaWithTLSAndSASL(t *testing.T) {
+	cert, err := tls.X509KeyPair(localhostCert, localhostKey)
+	require.NoError(t, err, "failed to load key pair")
+
+	ctx := context.Background()
+
+	container, err := RunContainer(ctx,
+		WithTLS(localhostCert, localhostKey),
+		WithEnableSASL(),
+		WithEnableKafkaAuthorization(),
+		WithNewServiceAccount("superuser-1", "test"),
+		WithSuperusers("superuser-1"),
+	)
+	require.NoError(t, err)
```

Comment:
Without the change it would fail here because [this request](https://github.com/testcontainers/testcontainers-go/pull/2140/files#diff-49a855c9cbf11dab0ed9ff52eed0ee96bf141661354e1123fa49cb9e238fc4cbL203) failed